src/components/AvatarGroup/styled/avatar.js

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+import Avatar from '../../Avatar';
+
+const StyledAvatar = styled(Avatar)`
+    border: 2px solid white;
+    ${props =>
+        props.zIndex &&
+        `
+        z-index: ${props.zIndex};
+        `}
+    ${props =>
+        props.size === 'large' &&
+        `
+            margin-left: -1.6rem;
```

Comment:
We should use a css selector to apply `margin-left` for all avatars except the first.
If we use the margin-left for all avatars we will have problems using it in a layout